src/convnets/inceptions/inceptionv3.jl

```diff
@@ -33,10 +33,10 @@ Create an Inception-v3 style-B module
   - `inplanes`: number of input feature maps
 """
 function inceptionv3_b(inplanes)
-    branch3x3_1 = Chain(conv_norm((3, 3), inplanes, 384; stride = 2))
-    branch3x3_2 = Chain(conv_norm((1, 1), inplanes, 64)...,
-                        conv_norm((3, 3), 64, 96; pad = 1)...,
-                        conv_norm((3, 3), 96, 96; stride = 2)...)
+    branch3x3_1 = Chain(basic_conv_bn((3, 3), inplanes, 384; stride = 2)...)
```

Comment:
If the timing difference is inference, could this `...` have made the difference? There are a couple places where this occurs, and mixing `Chain(::Vector)` and `Chain(::Tuple)` sounds like a recipe of problems.